lib/resources/example_json/dstu2_examples_schedule.rb

```diff
@@ -5,35 +5,35 @@ module Resources
       "resourceType": "Schedule",
       "id": "21265426-633867-3121665-0",
       "text": {
-        "status": "generated",
-        "div": "&lt;div&gt;&lt;p&gt;&lt;b&gt;Schedule&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Type&lt;/b&gt;: Same Day&lt;/p&gt;&lt;p&gt;&lt;b&gt;Practitioner&lt;/b&gt;: Song, River&lt;/p&gt;&lt;p&gt;&lt;b&gt;Location&lt;/b&gt;: Baseline West&lt;/p&gt;&lt;/div&gt;"
+          "status": "extensions",
```

Comment:
Formatting for our examples is 1 tab (2 spaces), not 2 tabs. 

Apply throughout

Reply:
Fix, my vscode got the formatting all messed up: https://github.com/cerner/fhir.cerner.com/pull/364/commits/1ba16fe403692544611158def892c63d53243121#diff-3b68091084d419442eda94be9a6438bdR8